### PR TITLE
:bug: Source Monday: Add new streams and update old 

### LINKED ISSUE #26672: Source Monday: new stream Workspaces
### What area the feature impact?

Connectors

## Tell us about the problem you're trying to solve
[Workspaces](https://developer.monday.com/api-reference/docs/workspaces)
monday.com [workspaces](https://support.monday.com/hc/en-us/articles/360010785460-The-Workspaces) are used by teams to manage their accounts by departments, teams, or projects. Workspaces can contain boards, dashboards, and folders to help you stay organized.

As a developer working with monday.com, it is important to familiarize yourself with the workspaces API so you know how to access workspace data. This document will walk you through the available queries and mutations to read and modify the workspaces object via the API.
Required scope: `workspaces:read`

## Describe the solution you’d like
Research pros and cons for implementing this stream

### PR DESCRIPTION
**Pull request by tolik0**
## What
Add new streams: Tags, Workspaces. Add new fields for existing streams: Columns, Column values, Files, Board views.
Issues: #26664,  #26667, #26668, #26670, #26671,  #26672, 

## How
Add new streams for tags and workspaces (schemas folder).
Extend **_Items_** stream for additional column values and files fields, the same for files in _**Updates**_ stream, also add new fields for columns and board views in **_Boards_** stream.
Minor updates into **_Boards_** stream.

## 🚨 User Impact 🚨 Breaking Changes
Schema changes:
- Stream **Boards**: change deprecated "_owner_" property to "_owners_"", add new property "_creator_", add new fields for "_columns_" and "_views_'
- Stream **Items**: add new fields for "_assets_" and "_column_values_" properties
- Stream **Updates**: new fields for "_assets_"
- Add new streams: **Tags**, **Workspaces**

### CODE REVIEW

**Comment by octavia-squidington-iii** [comment]
##  source-monday test report (commit [`669d722101`](https://github.com/airbytehq/airbyte/pull/27410/commits/669d722101c55bb27651f470729495fc12fcc347)) - ❌

⏲️  Total pipeline duration: 510 seconds

| Step                                                                 | Result   |
|:---------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-monday/metadata.yaml | ✅        |
| Connector version semver check.                                      | ✅        |
| Connector version increment check.                                   | ✅        |
| QA checks                                                            | ✅        |
| Code format checks                                                   | ✅        |
| Connector package install                                            | ✅        |
| Build source-monday docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                           | ✅        |
| Acceptance tests                                                     | ❌        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5332349810)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-monday test
```

**Comment by octavia-squidington-iii** [comment]
##  source-monday test report (commit [`d9d7d79be0`](https://github.com/airbytehq/airbyte/pull/27410/commits/d9d7d79be0a558a4ad6791f43684cf8da8d14b7c)) - ❌

⏲️  Total pipeline duration: 32 seconds

| Step                                                                 | Result   |
|:---------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-monday/metadata.yaml | ✅        |
| Connector version semver check.                                      | ✅        |
| Connector version increment check.                                   | ✅        |
| QA checks                                                            | ✅        |
| Code format checks                                                   | ✅        |
| Connector package install                                            | ❌        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5333047726)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-monday test
```

**Comment by octavia-squidington-iii** [comment]
##  source-monday test report (commit [`d9d7d79be0`](https://github.com/airbytehq/airbyte/pull/27410/commits/d9d7d79be0a558a4ad6791f43684cf8da8d14b7c)) - ✅

⏲️  Total pipeline duration: 557 seconds

| Step                                                                 | Result   |
|:---------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-monday/metadata.yaml | ✅        |
| Connector version semver check.                                      | ✅        |
| Connector version increment check.                                   | ✅        |
| QA checks                                                            | ✅        |
| Code format checks                                                   | ✅        |
| Connector package install                                            | ✅        |
| Build source-monday docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                           | ✅        |
| Acceptance tests                                                     | ✅        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5333178916)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-monday test
```

**Comment by octavia-squidington-iii** [comment]
##  source-monday test report (commit [`15870dda6c`](https://github.com/airbytehq/airbyte/pull/27410/commits/15870dda6c1515565ea525c1cdfab4f70df68eda)) - ✅

⏲️  Total pipeline duration: 450 seconds

| Step                                                                 | Result   |
|:---------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-monday/metadata.yaml | ✅        |
| Connector version semver check.                                      | ✅        |
| Connector version increment check.                                   | ✅        |
| QA checks                                                            | ✅        |
| Code format checks                                                   | ✅        |
| Connector package install                                            | ✅        |
| Build source-monday docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                           | ✅        |
| Acceptance tests                                                     | ✅        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5333376430)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-monday test
```

File: airbyte-integrations/connectors/source-monday/Dockerfile [hunk 1]
```diff
@@ -34,5 +34,5 @@ COPY source_monday ./source_monday
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.2.6
+LABEL io.airbyte.version=0.2.7
```
Comment (davydov-d) [inline airbyte-integrations/connectors/source-monday/Dockerfile:37]: this should be a major version bump - 1.0.0
https://docs.airbyte.com/contributing-to-airbyte/#semantic-versioning-for-connectors

**Review by davydov-d — APPROVED** [review]

File: airbyte-integrations/connectors/source-monday/Dockerfile (hunk 1 above)
Comment (tolik0) [reply airbyte-integrations/connectors/source-monday/Dockerfile]: Fixed

**Comment by octavia-squidington-iii** [comment]
##  source-monday test report (commit [`f6ed589ea9`](https://github.com/airbytehq/airbyte/pull/27410/commits/f6ed589ea93b7b3ac6bcade8c4448e5111e16b3d)) - ✅

⏲️  Total pipeline duration: 502 seconds

| Step                                                                 | Result   |
|:---------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-monday/metadata.yaml | ✅        |
| Connector version semver check.                                      | ✅        |
| Connector version increment check.                                   | ✅        |
| QA checks                                                            | ✅        |
| Code format checks                                                   | ✅        |
| Connector package install                                            | ✅        |
| Build source-monday docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                           | ✅        |
| Acceptance tests                                                     | ✅        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5335427968)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-monday test
```

**Review by bazarnov — APPROVED** [review]

**Review by lazebnyi — APPROVED** [review]

**Review by brianjlai — APPROVED** [review]
🚢

**Comment by octavia-squidington-iii** [comment]
##  source-monday test report (commit [`b05718558b`](https://github.com/airbytehq/airbyte/pull/27410/commits/b05718558bfe4502d975bb065a709beb9053946d)) - ✅

⏲️  Total pipeline duration: 477 seconds

| Step                                                                 | Result   |
|:---------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-monday/metadata.yaml | ✅        |
| Connector version semver check.                                      | ✅        |
| Connector version increment check.                                   | ✅        |
| QA checks                                                            | ✅        |
| Code format checks                                                   | ✅        |
| Connector package install                                            | ✅        |
| Build source-monday docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                           | ✅        |
| Acceptance tests                                                     | ✅        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5355861625)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-monday test
```

**Comment by jnr0790** [comment]
@tolik0 let's schedule this for July 11th at 4 pm UTC. I'll get comms sent to the users today.

**Comment by octavia-squidington-iii** [comment]
##  source-monday test report (commit [`3c70a710ed`](https://github.com/airbytehq/airbyte/pull/27410/commits/3c70a710ed5c2f9ff0bdcd61913861530ae1840c)) - ✅

⏲️  Total pipeline duration: 497 seconds

| Step                                                                 | Result   |
|:---------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-monday/metadata.yaml | ✅        |
| Connector version semver check.                                      | ✅        |
| Connector version increment check.                                   | ✅        |
| QA checks                                                            | ✅        |
| Code format checks                                                   | ✅        |
| Connector package install                                            | ✅        |
| Build source-monday docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                           | ✅        |
| Acceptance tests                                                     | ✅        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5389922931)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-monday test
```

**Comment by octavia-squidington-iii** [comment]
##  source-monday test report (commit [`f8000a8b4a`](https://github.com/airbytehq/airbyte/pull/27410/commits/f8000a8b4a29a2e334bddea33fdb7598596dd9d0)) - ✅

⏲️  Total pipeline duration: 456 seconds

| Step                                                                 | Result   |
|:---------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-monday/metadata.yaml | ✅        |
| Connector version semver check.                                      | ✅        |
| Connector version increment check.                                   | ✅        |
| QA checks                                                            | ✅        |
| Code format checks                                                   | ✅        |
| Connector package install                                            | ✅        |
| Build source-monday docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                           | ✅        |
| Acceptance tests                                                     | ✅        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5391311316)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-monday test
```

**Comment by jnr0790** [comment]
just a reminder to get this pushed

**Pull request merged**